src/games/strategy/triplea/util/TransportUtils.java

```diff
@@ -158,13 +151,18 @@ public static int getTransportCost(final Collection<Unit> units) {
     return mapping;
   }
 
-  private static List<Unit> sortByTransportCapacityAscending(final Collection<Unit> transports) {
+  private static List<Unit> sortByTransportCapacityAscendingThenMovesDescending(final Collection<Unit> transports) {
     final Comparator<Unit> transportCapacityComparator = new Comparator<Unit>() {
       @Override
       public int compare(final Unit o1, final Unit o2) {
         final int capacityLeft1 = TransportTracker.getAvailableCapacity(o1);
         final int capacityLeft2 = TransportTracker.getAvailableCapacity(o2);
-        return capacityLeft1 - capacityLeft2;
+        if (capacityLeft1 != capacityLeft2) {
+          return Integer.compare(capacityLeft1, capacityLeft2);
```

Comment:
@DanVanAtta I think you are mistaken as the ordering based on capacity isn't change: https://docs.oracle.com/javase/7/docs/api/java/lang/Integer.html#compare(int,%20int)